classes/WP_CLI/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -175,11 +171,9 @@ protected function print_total_batches( $batches_completed ) {
 	/**
 	 * Convert an exception into a WP CLI error.
 	 *
-	 * @author Jeremy Pry
-	 *
 	 * @param Exception $e The error object.
 	 *
-	 * @throws \WP_CLI\ExitException
+	 * @throws \WP_ClI\ExitException Under some conditions WP CLI may throw an exception.
```

Comment:
Applying a further adjustment (minor).

```suggestion
	 * @throws \WP_CLI\ExitException Under some conditions WP CLI may throw an exception.
```